challenges/type-unions/tests.ts

```diff
@@ -0,0 +1,117 @@
+import { Expect } from 'type-testing';
+import { expect } from 'jest';
```

Comment:
@dimitropoulos @ggrandi @bautistaaa is this technically possible with our current implementation?

Reply:
glad you asked! I don't think it is yet, but to me it's among a short list of things that are absolutely critical.  It doesn't even have to be jest or whatever.  but being able to run actual javascript (and test it) is one of the biggest problems with type-challenges because it locks the entire project into the type realm _only_.  there are lots of important grounds to cover (i.e. practical things like we all do in our daily life with TypeScript) that you simply cannot do with types alone.  e.g. a challenge I'm planning on how to make generic constraints for functions (a pretty important skill) really doesn't work unless you can write a function.

as for the particular implementation here, again, doesn't matter.  I'd guess that we'll need to do some kind of `describe` block first or something if we want to accomplish actually running the tests.

Reply:
oh, another thought, I haven't looked into this particular space of the testing world in a bit (assertion libraries) but https://github.com/power-assert-js/power-assert used to be a really fantastic choice (and maybe it still is).  it has no API, which makes it really easy to use.  You just construct your assertions so that they return booleans.

Reply:
we’re going to need some time to research the js capability 

we’d need a console to output logs maybe? but mainly sounds like you just need those tests to pass

this potentially might take some time. is it possible to write a challenge within these constraints for now 

ill prioritize this research this week

Reply:
> is it possible to write a challenge within these constraints for now

In most cases, yes.  We can just comment out this one's usage of `expect` for now